apps/condo/domains/onboarding/components/TourPage/AppCards.tsx

```diff
@@ -42,12 +42,26 @@ export const TechnicAppCard = () => {
     const TechnicAppCardAdminLabel = intl.formatMessage({ id: 'tour.technicAppCard.radio.label.admin' })
     const TechnicAppCardTechnicLabel = intl.formatMessage({ id: 'tour.technicAppCard.radio.label.technic' })
     const TechnicAppCardSecurityLabel = intl.formatMessage({ id: 'tour.technicAppCard.radio.label.security' })
+    const ChantBotMessage = intl.formatMessage({ id: 'ChatBot' }).toLowerCase()
+    const OpenChatBotMessage = intl.formatMessage({ id: 'tour.technicAppCard.download.linkToEmployeeBot' })
 
     const [activeModal, setActiveModal] = useState<ActiveModalType>()
     const [radioValue, setRadioValue] = useState<RadioOptionType>('admin')
 
     const TechnicAppCardText = intl.formatMessage({ id: `tour.technicAppCard.radio.text.${radioValue}` })
 
+    const showLinkToEmployeeBot = !!telegramEmployeeBotName && (radioValue === 'admin' || radioValue === 'technic')
+    const linkToEmployeeBot = useMemo(() => telegramEmployeeBotName ? (
+        <Typography.Link
+            href={`https://t.me/${telegramEmployeeBotName}`}
+            target='_blank'
+            id={`employee-telegram-bot-from-onboarding-by-technic-app-card-as-${radioValue}`}
+        >
+            {ChantBotMessage}
```

Comment:
Hey, it seems like there are a typo here :-)

Reply:
You're right, I will fix it)